### ISSUE TITLE
Recommended version of Python for medspaCy 1.0.0

### ISSUE THREAD

**Issue by RonWilkinson** [medspacy/medspacy#160]
I have been running medspacy 0.2.0 in a Python 3.7 environment, because I found that higher versions of Python broke it.  After installing medspacy 1.0.0 in that environment though, I get an error importing medspacy, because it uses typing.Literal, which was implemented in Python 3.8.  I am happy to create a new environment for medspacy, but want to make sure I create one on which it is well tested.

**Comment by burgersmoke** [comment]
Great question @RonWilkinson .  You are correct that 3,7 is no longer supported.

So answer your questions, our continuous integration of unit tests, etc is currently running Python 3.8.

Meanwhile, many of the medspacy developers have been using Python 3.9.

Does this answer your question?

**Comment by RonWilkinson** [comment]
Yes.  I will be slightly adventurous and use a 3.9 environment like the medspacy developers rather than 3.8, and I will not try to use a 3.10.7 environment, which is the most current full release of Python as of Sept. 6, 2022.  Thanks.

**Comment by burgersmoke** [comment]
Sounds great.  I think 3.9 is a good place to be for now.

**Comment by andyjessen** [comment]
@RonWilkinson  - Thanks for raising this issue. I encountered the same problem today.

**Comment by burgersmoke** [comment]
Agreed.  This PR from @andyjessen  looks like a good one so that we can prevent issues like this.  I believe we'll be merging this in soon and likely do a minor release.
